core/primitives/src/views.rs

```diff
@@ -297,6 +297,9 @@ pub struct StatusSyncInfo {
     pub latest_state_root: CryptoHash,
     pub latest_block_time: DateTime<Utc>,
     pub syncing: bool,
+    pub earliest_block_hash: Option<CryptoHash>,
+    pub earliest_block_height: Option<BlockHeight>,
+    pub earliest_block_time: Option<DateTime<Utc>>,
```

Comment:
We probably don't need to have these two because depending on whether the node is archival, the earliest block can be very different whereas latest block is likely the same across different types of nodes. There is, in some sense, no point in exposing earliest block height and earliest block time in the response. They can be queried easily if one knows the earliest block hash.

Reply:
I think exposing these two fields slightly improves the usability. If a user has more than 1 node, querying the block hashes can be frustrating.